src/main/java/com/exadel/discountwebapp/vendor/service/VendorService.java

```diff
@@ -45,7 +45,8 @@ public VendorResponseVO findByTitle(String title) {
 
     @Transactional(propagation = Propagation.REQUIRED)
     public VendorResponseVO create(VendorRequestVO request) {
-        vendorEmailValidator.validate(request);
+        vendorValidator.checkDuplicateEmail(request);
+        vendorValidator.checkDuplicatePhones(request);
```

Comment:
these 2 validations can be performed inside of one "vendorValidator.validateForCreate(request)" method

Reply:
done